src/fa/game_session.py

```diff
@@ -177,13 +183,46 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        self._add_to_full_gamelog()
+
+    def _limit_replayid_gamelogs(self):
+        nb_replayid_files = 0
+        files = os.listdir(util.LOG_DIR)
+        files = map(lambda f: os.path.join(util.LOG_DIR, f), files)
+        files = sorted(files, key = os.path.getmtime)
+        files = list(map(os.path.basename, files))
+        replay_files = [e for e in files if "Replayid" in e]
+        while len(replay_files) >= util.MAX_NUMBER_REPLAYID_LOG_FILE:
+            os.remove(os.path.join(util.LOG_DIR,replay_files[0]))
+            replay_files.pop(0)
+
+    def _add_to_full_gamelog(self):
+        if os.path.isfile(util.LOG_FILE_GAME):
+            with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME + '.full.log', 'a+') as dst: dst.write(src.read())
+
+    def _clear_initial_gamelog(self):
+        with open(util.LOG_FILE_GAME, 'w') as src : pass
```

Comment:
Is we explicitly make it a temporary file, we can just remove it here.

Reply:
You mean that once I renamed all the files as you said, I can just remove this function ?